Bottom: cb9640ff81712b84bd62074536a082d4d13b88af
Top:    821f217b6d02cdabf3e98722b9321042f94054db
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-02-14 19:10:32 +0000

Refresh of add-action-to-break-elements

---

diff --git a/src/action.c b/src/action.c
index 962ab27..828f2f1 100644
--- a/src/action.c
+++ b/src/action.c
@@ -8030,7 +8030,8 @@ This smashes the given object into pieces in-place on the PCB.
 %end-doc */
 
 
-static void SmashElement (ElementType *element)
+static void
+SmashElement (ElementType *element)
 {
   Cardinal group;
   LayerType *top_copper, *bottom_copper;
